.github/dependabot.yml

```diff
@@ -12,6 +12,15 @@ updates:
         patterns:
           - "*/pipeline-Nextflow-config"
           - "*/pipeline-Nextflow-module"
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-name: "external/pipeline-*"
```

Comment:
**question (non-blocking):** Are you including this for future-proofing? All submodules in the repo currently match this glob.

Reply:
Partially for any potential additions and also to make maintaining easier by including all `pipeline-*` and the ignoring any specific ones we want from the rules